docker-compose.yml

```diff
@@ -8,6 +8,34 @@ services:
     hostname: coreapi-postgres
     environment:
       POSTGRESQL_USER: coreapi
-      POSTGRESQL_PASSWORD: coreapipostgres
+      POSTGRESQL_PASSWORD: coreapi
       POSTGRESQL_DATABASE: coreapi
-    container_name: coreapi-postgres
\ No newline at end of file
+    container_name: coreapi-postgres
+
+  alembic_migrations:
+    build:
+      context: .
+      dockerfile: Dockerfile.migrations
+    network_mode: bridge
+    hostname: alembic_migrations
+    links:
+      - postgres
+    container_name: alembic_migrations
+
+  f8a_ingestion:
+    build: .
+    network_mode: bridge
+    ports:
+     - "5000:5000"
+    hostname: f8a_ingestion
+    environment:
+      INGESTION_API_SERVICE_PORT: 5000
+      INGESTION_API_SERVICE_TIMEOUT: 3000
+      POSTGRESQL_USER: coreapi
+      POSTGRES_PASSWORD: coreapi
```

Comment:
Is this correct? Why do we have `POSTGRESQL_USER`, but `POSTGRES_PASSWORD`? Note the missing `QL` in the password env. variable.